pycbc/workflow/injection.py

```diff
@@ -243,12 +243,16 @@ def setup_injection_workflow(workflow, output_dir=None,
                 'tags': curr_tags
             }
             injection_path = workflow.cp.get_opt_tags(
-                "workflow-injections",
+                f"workflow-{inj_section_name}",
                 "injections-pregenerated-file",
                 curr_tags
             )
             curr_file = resolve_url_to_file(injection_path, attrs=file_attrs)
             inj_files.append(curr_file)
+        elif injection_method == "IN_FRAME":
+            # IN_FRAME means that we will supply frames with the appropriate
+            # injections unknown to the analysis
+            inj_files.append(None)
```

Comment:
My only substantial comment is that having a File be None is a hack. If anyone accidentally sent this file to a job, pegasus is going to [rightly] break. There doesn't seem to be a good way to handle this though ...

Reply:
Yeah, though people _shouldn't_ do that, there wont be a nice error message to say what they have done which is wrong either